tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -409,47 +477,22 @@ def test_ln_raises_exception_when_target_path_does_not_exist(simulation_file, fs
     with pytest.raises(FileNotFoundError):
         simulation_file.ln("computer", "work")
 
-
 # ========== end of ln() tests ==========
 
 
-def test_check_path_in_computer_is_abs(fs):
+def test_check_path_in_computer_is_abs(simulation_file, fs):
     """
     Tests that ``esm_parser.user_error`` is used when the ``path_in_computer``
     is not absolute
     """
+    simulation_file.path_in_computer = Path("foo/bar")
+    with pytest.raises(SystemExit):
+        simulation_file._check_path_in_computer_is_abs()
 
-    dummy_config = """
-    general:
-        thisrun_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101"
-        exp_dir: "/work/ollie/pgierz/some_exp"
-        thisrun_work_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101/work"
-    echam:
-        files:
-            jan_surf:
-                type: input
-                name_in_computer: T63CORE2_jan_surf.nc
-                name_in_work: unit.24
-                path_in_computer: pool/ECHAM/T63
-        experiment_input_dir: /work/ollie/pgierz/some_exp/input/echam
-        thisrun_input_dir: /work/ollie/pgierz/some_exp/run_20010101-20010101/input/echam
-    """
-    config = yaml.safe_load(dummy_config)
-
-    # Captures output (i.e. the user-friendly error)
-    with Capturing() as output:
-        with pytest.raises(SystemExit) as error:
-            sim_file = esm_runscripts.filedicts.SimulationFile(
-                config, "echam.files.jan_surf"
-            )
 
-    # error needs to occur as the path is not absolute
-    assert any(["ERROR: File Dictionaries" in line for line in output])
```

Comment:
Please do not remove this part. I think @mandresm has more plans for it.

Reply:
I brought it back but still keep the MUT because this is better for unit testing.

Reply:
My comment only referred to that line, not the entire method